### DISCUSSION TITLE
Converting from org mode: Internal Links not Handled?

### DISCUSSION THREAD

**Discussion by beetleb** [jgm/pandoc#10158] [Q&A]
I have an org mode document with an [internal link](https://orgmode.org/manual/Internal-Links.html) of the form:

```
[[*Header Name][description]]
```

This should link to the node whose title is "Header Name", but the output from pandoc is:

```
*Header Name*
```

I tried with HTML and restructured text as outputs.

(Note that the `[[#target][link]]` format works).

**Comment by jgm** [comment]
@tarleb should know more about this.  If we don't currently support this, we should.

**Reply by tarleb** [reply]
I'll give it a closer look this weekend. Relevant issue: #6916
